data/mods/DinoMod/monsters/zed-dinosaur.json

```
@@ -253,6 +220,30 @@
     "description": "The shuffling corpse of what looks like a giant armadillo with peeling armored plates and black, glistening eyes.  Its tail ends in a massive spiked club of bone.",
     "upgrades": { "half_life": 14, "into": "mon_zankylosaurus_brute" }
   },
+  {
+    "type": "MONSTER",
+    "id": "mon_ziplodocus",
+    "name": { "str_sp": "Diplodocus zombie" },
```

```suggestion
    "name": { "str": "Diplodocus zombie" },
```
"str_sp" means the singular form is the same as the plural form.